doc/user/RouterConfiguration.md

```diff
@@ -923,7 +924,11 @@ Used to group requests when monitoring OTP.
         "f58761e5-8dd5-4940-a472-872f1236c596"
       ]
     }
-  ]
+  ],
+  "triasApi" : {
+    "hideFeedId" : true,
+    "hardcodedInputFeedId" : "nvbw"
+  }
```

Comment:
Should not be here

Reply:
b33f806535203214b22b2f5d057083ad55c6a669